src/services/fetcher.ts

```diff
@@ -0,0 +1,9 @@
+export const fetcher = async (path: string, section: string = "home") => {
+  const res = await fetch(
+    `${path}/svc/topstories/v2/${section}.json?api-key=${process.env.NEXT_PUBLIC_API_TOKEN}`
```

Comment:
Looks like path it's a constant like `NEXT_PUBLIC_API_TOKEN`. We do not need to pass it as argument.

Ideally you wanna do something like this:
Create file for constants, `src/constants/api.ts` with
```
export const BASE_PATH = process.env.NEXT_PUBLIC_BASE_PATH;
export const API_KEY = process.env.NEXT_PUBLIC_API_KEY;
```

And use this constants here.